tests/LengthConvertorTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+namespace ConvertorApp;
+class LengthConvertorTest extends \PHPUnit_Framework_TestCase {
+  /**
+	 *@test
+	 */
+	public function convertMeterToKilometer_function_can_validate_integer_value() {
+		$convert = new LengthConvertor();
+		$result  = $convert->convertMeterToKilometer(1);
+		$answer = $result-(0.001);
+		$this->assertLessThan(0.0001, $answer);
+	}
+	/**
+	 *@test
+	 * @expectedException InvalidArgumentException
+	 */
+  public function convertMeterToKilometer_function_should_throw_InvalidArgumentException_for_argument_which_Contain_string() {
+   $convert = new LengthConvertor();
+	 $convert->convertMeterToKilometer("abc");
+	}
+  /**
+	 *@test
+	 */
+  public function convertMeterToKilometer_function_can_validate_negative_integer_argument() {
+	 $convert = new LengthConvertor();
+	 $result  = $convert->convertMeterToKilometer(-1);
+	 $answer = $result-(-0.001);
```

Comment:
for string checking test cases function throws exception , then why we need assertion for that?